app/experimenter/nimbus-ui/src/types/getConfig.ts

```diff
@@ -0,0 +1,83 @@
+/* tslint:disable */
```

Comment:
So neat that it auto generates these types for us, I wanted to do this with the rest API and its auto generated openjson schema.